Terminal.Gui/Views/TableView/TableView.cs

```diff
@@ -315,16 +316,16 @@ private int GetHeaderHeightIfAny ()
 		/// <returns></returns>
 		private int GetHeaderHeight ()
 		{
-			int heightRequired = 0;
-			if (!Style.HideHeaders) {
-				heightRequired++;
+			if (Style.HideHeaders)
+				return 0;
```

Comment:
Please use brackets for all nested statements. Even single lines. Thanks.

Reply:
This was mooted, but thanks for the reminder.

Reply:
> This was mooted, but thanks for the reminder.

Not sure what you mean by `mooted`. 

Before I can merge this, I need you to add brackets to all `if` statements, even if just a single line.